modelforge/tests/helper_functions.py

```diff
@@ -130,7 +123,6 @@ def initialize_dataset(
     data = dataset(for_unit_testing=for_unit_testing)
     data_module = TorchDataModule(data, split_file=split_file)
     data_module.prepare_data()
-    data_module.setup(stage=mode)
     return data_module
 
 
```

Comment:
I think a renaming attempt went rouge: 

`preparePairlist_for_single_batch`

should be `prepare_pairlist_for_single_batch`.